lib/mini_fb.rb

```diff
@@ -740,12 +740,7 @@ def self.fetch(url, options={})
             end
             return params
         else
-            begin
-                res_hash = JSON.parse(resp.to_s)
-            rescue
-                # quick fix for things like stream.publish that don't return json
-                res_hash = JSON.parse("{\"response\": #{resp.to_s}}")
-            end
+            res_hash = JSON.parse(resp.to_s.size > 2 ? resp.to_s : {response: resp.to_s}.to_json)
```

Comment:
Yes, I did a test with a 201 and even if is a nil, nil.to_s returns "" so we should be okey